query-service-impl/src/test/resources/trino_service_request_handler.conf

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Not needed. If possible, can we remove this.

Reply:
removed